packages/commands/package.json

```diff
@@ -27,9 +27,12 @@
 	"react-native": "src/index",
 	"dependencies": {
 		"@babel/runtime": "^7.16.0",
+		"@wordpress/block-editor": "file:../block-editor",
```

Comment:
I think we should avoid having block editor as a dependency here. Maybe we can avoid using the Warning component here.